railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -70,9 +70,19 @@ RUN apt-get update -qq && \
 COPY --from=build /usr/local/bundle /usr/local/bundle
 COPY --from=build /rails /rails
 
+# add custom user
+RUN useradd ruby
```

Comment:
Not sure I like the name `ruby`. Might be confusing. I'd rather use something like `appuser`

Reply:
agree here